modules/user-data/variables.tf

```diff
@@ -66,3 +66,27 @@ variable "appi_connection_string" {
   description = "Connection string for Application Insights"
   type        = string
 }
+
+variable "appi_sampling_percentage" {
```

Comment:
We add such variables in the submodules all the thime but not all of them are available via the root module :thinking:

Reply:
yes, we need to make the available on root as well